src/Framework/TestCase.php

```diff
@@ -1876,6 +1882,23 @@ private function handleDependencies(): bool
         return true;
     }
 
+    private function markSkippedForNotSpecifyingDependency(): void
+    {
+        $this->status = BaseTestRunner::STATUS_SKIPPED;
+
+        $this->result->startTest($this);
+
+        $this->result->addError(
+            $this,
+            new SkippedTestError(
+                \sprintf('This method is invalid at depends annotation.')
```

Comment:
Perhaps this is better?

```suggestion
                \sprintf('This method has an invalid @depends annotation.')
```

What do you think?

Reply:
I thought about this but then I thought maybe the @depends keyword can change in the future than we have to change the message too!